bin/fusor-upgrade-qci

```diff
@@ -0,0 +1,192 @@
+    if [[ $REPLY =~ ^[Yy]$ ]]; then
+      echo -e "\n continuing... \n"
+    else
+      echo -e "\n\nUser ABORT! Not running the QCI upgrade script.. and exiting..\n"
+      exit
+    fi
+  fi
+
+  # install the sat6 repo
+  if [ ! -f "${SAT6_REPO_FILE_NAME}" ]; then
+    cat << EOF > ${SAT6_REPO_FILE_NAME}
+[${SAT6_REPO_ID}]
+name=Satellite 6
+baseurl=${SAT6_REPO_URL}
+enabled=1
+gpgcheck=0
+EOF
+  else
+    echo -e "[WARNING] '${SAT6_REPO_FILE_NAME}' exists! "
+    read -p "Do you want to continue? [y/N] " -n 1 -r
+    if [[ $REPLY =~ ^[Yy]$ ]]; then
+      echo -e "\n continuing... \n"
+    else
+      echo -e "\n\nUser ABORT! Not running the QCI upgrade script.. and exiting..\n"
+      exit
+    fi
+  fi
+
+  # enable the fusor repo and disable the sat6 repo for the QCI upgrade
+  yum-config-manager --enable ${FUSOR_REPO_ID}
+  yum-config-manager --disable ${SAT6_REPO_ID}
+  yum clean all
+
+  # upgrade selinux-policy to satisfy the Satellite upgrade requirement
+  yum upgrade selinux-policy -y
+  if [ "$?" -ne "0" ]; then
+    echo -e "[ERROR] upgrading selinux-policy\n"
+    exit
+  fi
+
+  # install yum-utils 
+  yum install yum-utils -y
+  if [ "$?" -ne "0" ]; then
+    echo -e "[ERROR] installing yum-utils\n"
+    exit
+  fi
+
+  # disable the fusor repo and enable the sat6 repo for the Satellite's upgrade
+  yum-config-manager --disable ${FUSOR_REPO_ID}
+  yum-config-manager --enable ${SAT6_REPO_ID}
+  yum clean all
+
+  # upgrade Satellite and other packages from the sat6 repo
+  yum update -y
+  if [ "$?" -ne "0" ]; then
+    echo -e "[ERROR] upgrading Satellite and its dependent packages!\n"
+    exit
+  fi
+
+  # enable the fusor repo and disable the sat6 repo for the QCI package upgrades
+  yum-config-manager --enable ${FUSOR_REPO_ID}
+  yum-config-manager --disable ${SAT6_REPO_ID}
+  yum clean all
+
+  yum upgrade qci -y
+  if [ "$?" -ne "0" ]; then
+    echo -e "[ERROR] upgrading ansible-ocp\n"
+    exit
+  fi
+
+  yum upgrade fusor-installer -y
+  if [ "$?" -ne "0" ]; then
+    echo -e "[ERROR] upgrading fusor-installer\n"
+    exit
+  fi
+
+  yum upgrade fusor-initial-setup -y
+  if [ "$?" -ne "0" ]; then
+    echo -e "[ERROR] upgrading fusor-initial-setup\n"
+    exit
+  fi
+
+  yum upgrade fusor-selinux -y
+  if [ "$?" -ne "0" ]; then
+    echo -e "[ERROR] upgrading fusor-selinux\n"
+    exit
+  fi
+
+  yum upgrade fusor_ovirt -y
+  if [ "$?" -ne "0" ]; then
+    echo -e "[ERROR] upgrading fusor_ovirt\n"
+    exit
+  fi
+
+  yum upgrade fusor-utils -y
+  if [ "$?" -ne "0" ]; then
+    echo -e "[ERROR] upgrading fusor-utils\n"
+    exit
+  fi
+
+  yum upgrade qci-sos-plugin -y
+  if [ "$?" -ne "0" ]; then
+    echo -e "[ERROR] upgrading qci-sos-plugin\n"
+    exit
+  fi
+
+  yum install ansible-ovirt -y
+  if [ "$?" -ne "0" ]; then
+    echo -e "[ERROR] installing ansible-ovirt\n"
+    exit
+  fi
+
+  yum install ansible-ocp -y
+  if [ "$?" -ne "0" ]; then
+    echo -e "[ERROR] installing ansible-ocp\n"
+    exit
+  fi
+
```

Comment:
I would remove the prior lines for setting up repos and running package install commands.
Document what the steps are a user should follow:
 1) Enable new repos
 2) run "yum upgrade *"